ext/dcos-installer/dcos_installer/cli.py

```diff
@@ -50,7 +50,15 @@ def print_usage():
 Install Mesosophere's Data Center Operating System
 
 dcos_installer [-h] [-f LOG_FILE] [--hash-password HASH_PASSWORD] [-v]
```

Comment:
`[--cli-telemetry-disabled]` should go here based on how the argument is added (it's not part of the mutually exclusive set).